scripts/test_python.js

```diff
@@ -41,11 +41,11 @@ const pytest_client_mode = IS_DOCKER => {
     if (IS_DOCKER) {
         return bash`${docker(IMAGE)} bash -c "cd \
             python/perspective && TZ=UTC ${PYTHON} -m pytest \
-            ${VERBOSE ? "-vv" : ""} --noconftest 
+            ${VERBOSE ? "-vv --full-trace" : ""} --noconftest 
```

Comment:
Is this necessary for typical builds or is this debug cruft?  While I'm sure the info is useful in some cases, long stdout make Azure cry.

Reply:
This is only for `--debug` - it should have no effect on regular builds or Azure.